security-jwt/src/main/java/io/micronaut/security/token/jwt/validator/JwtTokenValidatorUtils.java

```diff
@@ -112,6 +112,34 @@ private JwtTokenValidatorUtils() {
         return Optional.empty();
     }
 
+    /**
+     *
+     * Validates a Signed JWT signature against a signature configuration.
+     *
+     * @param signedJWT a Signed JWT Token
+     * @param config The signature configuration
+     * @return empty if signature validation fails
+     */
+    public static Optional<JWT> verifySignedJWTWithSignatureConfiguration(SignatureConfiguration config, SignedJWT signedJWT) {
```

Comment:
Make this private

Reply:
I think it is valuable to have it public to allow people to call it.